lib/bundler/cli.rb

```diff
@@ -474,6 +474,24 @@ def add(name, version = nil, *gems)
       Inject.new(options, name, version, gems).run
     end
 
+    desc "doctor [OPTIONS]", "Checks the bundle for common problems"
+    long_desc <<-D
+      Doctor scans the OS dependencies of each of the gems requested in the Gemfile. If
+      missing dependencies are detected, Bundler prints them and exits status 1.
+      Otherwise, Bundler prints a success message and exits with a status of 0.
+
+      The bundle's gem dependencies must all be installed to run this command; if
+      they are not, Bundler prints an error message and exits with a status of 2.
```

Comment:
I _think_ that I have just cancelled the imminent merge of this PR. Go ahead and push another commit, and I'll kick off another build to get a final merge with green tests. Thanks for following up!